src/phoenix/experimental/evals/templates/default_templates.py

```diff
@@ -38,3 +38,23 @@
     # Answer: {response}
     Is the answer above factual or hallucinated based on the query and reference text?
 """
+
+CODE_READABILITY_PROMPT_TEMPLATE_STR = """
```

Comment:
Can you also add a map called `CODE_READABILITY_PROMPT_OUTPUT_MAP`? It should be used as a mapping to transform the values in the "readability" column from booleans to string values.

See Nate's PR for reference: https://github.com/Arize-ai/phoenix/pull/1338/files

Reply:
It should look something like:

```python
CODE_READABILITY_PROMPT_OUTPUT_MAP = {
    True: "readable", False: "unreadable"
}
```

Reply:
See this comment for context: https://github.com/Arize-ai/phoenix/pull/1338#issuecomment-1720383668